components/dashboard/src/admin/ProjectsSearch.tsx

```diff
@@ -0,0 +1,132 @@
+                }
+            }
+        })();
+    }, [currentProject])
+
+    if (!user || !user?.rolesOrPermissions?.includes('admin')) {
+        return <Redirect to="/" />
+    }
+
+    if (currentProject) {
+        return <ProjectDetail project={currentProject} owner={currentProjectOwner} />;
+    }
+
+    const search = async () => {
+        setSearching(true);
+        try {
+            const result = await getGitpodService().server.adminGetProjectsBySearchTerm({
+                searchTerm,
+                limit: 50,
+                orderBy: 'creationTime',
+                offset: 0,
+                orderDir: "desc"
+            })
+            setSearchResult(result);
+        } finally {
+            setSearching(false);
+        }
+    }
+
+    return <>
+        <div className="pt-8 flex">
+            <div className="flex justify-between w-full">
+                <div className="flex">
+                    <div className="py-4">
+                        <svg className={searching ? 'animate-spin' : ''} width="16" height="16" fill="none" xmlns="http://www.w3.org/2000/svg">
+                            <path fillRule="evenodd" clipRule="evenodd" d="M6 2a4 4 0 100 8 4 4 0 000-8zM0 6a6 6 0 1110.89 3.477l4.817 4.816a1 1 0 01-1.414 1.414l-4.816-4.816A6 6 0 010 6z" fill="#A8A29E" />
+                        </svg>
+                    </div>
+                    <input type="search" placeholder="Search Projects" onKeyDown={(k) => k.key === 'Enter' && search()} onChange={(v) => { setSearchTerm(v.target.value) }} />
+                </div>
+                <button disabled={searching} onClick={search}>Search</button>
+            </div>
+        </div>
+        <div className="flex flex-col space-y-2">
+            <div className="px-6 py-3 flex justify-between text-sm text-gray-400 border-t border-b border-gray-200 dark:border-gray-800 mb-2">
+                <div className="w-4/12">Name</div>
+                <div className="w-6/12">Clone URL</div>
+                <div className="w-2/12">Created</div>
+            </div>
+            {searchResult.rows.map(project => <ProjectResultItem project={project} />)}
+        </div>
+    </>
+
+    function ProjectResultItem(p: { project: Project }) {
+        return (
+            <Link key={'pr-' + p.project.name} to={'/admin/projects/' + p.project.id} data-analytics='{"button_type":"sidebar_menu"}'>
+                <div className="rounded-xl whitespace-nowrap flex py-6 px-6 w-full justify-between hover:bg-gray-100 dark:hover:bg-gray-800 focus:bg-gitpod-kumquat-light group">
+                    <div className="flex flex-col w-4/12 truncate">
+                        <div className="font-medium text-gray-800 dark:text-gray-100 truncate">{p.project.name}</div>
+                    </div>
+                    <div className="flex flex-col w-6/12 truncate">
+                        <div className="font-medium text-gray-800 dark:text-gray-100 truncate">{p.project.cloneUrl}</div>
```

Comment:
**suggestion(non-blocking)**: What do you think of toning down the font weight and font color here to help with information hierarchy as we do with other similar lists?

```suggestion
                        <div className="text-gray-500 dark:text-gray-100 truncate">{p.project.cloneUrl}</div>
```

| BEFORE | AFTER |
|-|-|
| <img width="1170" alt="list-before" src="https://user-images.githubusercontent.com/120486/151969337-ea13e5f7-ee16-450a-b429-5efa610820af.png"> | <img width="1170" alt="list-after" src="https://user-images.githubusercontent.com/120486/151969342-ca2ca9a8-663e-4ab1-a40a-a42604970b23.png"> |